CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to Tangle-Accelerator
```

Comment:
`Tangle-accelerator`

Reply:
Jserv said use uppercase

Reply:
But our entire project uses `Tangle-accelerator` :sweat_smile:

Reply:
@jkrvivian is correct.

Reply:
OK let me change it